bin/pycbc_optimize_snr

```diff
@@ -27,6 +29,12 @@ from pycbc.psd import interpolate
 Nfeval = 0
 start_time = time.time()
 
+# https://pynative.com/python-generate-random-string/
+def random_string(stringLength=10):
```

Comment:
An identical function is defined in `pycbc_live`. Please move both definitions to a pycbc module (for example `pycbc/__init__.py`) and call those from the two locations.